diff --git a/configure b/configure
index 505767cede55..4074a1e2f765 100755
--- a/configure
+++ b/configure
@@ -753,8 +753,8 @@ case "$CFG_RELEASE_CHANNEL" in
           *-pc-windows-gnu)
             ;;
           *)
-            CFG_ENABLE_DEBUGINFO_LINES=1
-            CFG_ENABLE_DEBUGINFO_ONLY_STD=1
+            # CFG_ENABLE_DEBUGINFO_LINES=1
+            # CFG_ENABLE_DEBUGINFO_ONLY_STD=1
             ;;
         esac
 
@@ -765,8 +765,8 @@ case "$CFG_RELEASE_CHANNEL" in
           *-pc-windows-gnu)
             ;;
           *)
-            CFG_ENABLE_DEBUGINFO_LINES=1
-            CFG_ENABLE_DEBUGINFO_ONLY_STD=1
+            # CFG_ENABLE_DEBUGINFO_LINES=1
+            # CFG_ENABLE_DEBUGINFO_ONLY_STD=1
             ;;
         esac
 	;;
